pull_request_template.md

```diff
@@ -1,6 +1,23 @@
 # Purpose of this PR
 
+
+## Type of change
+
+- [ ] Bug fix 
+- [ ] Refactoring
+- [ ] New feature 
+- [ ] Minor change
+- [ ] Fundamental change
+- [ ] This change requires a documentation update
+
+
 ## Checklist:
-* updated in code documentation
-* adjusted reporting if needed
-* checked that REMIND still compiles
+
+- [ ] My code follows the coding ettiquette
```

Comment:
I think the "spelling etiquette" is with one t in etiquette

Reply:
:-)